pipelines/operator-hosted-pipeline.yml

```diff
@@ -0,0 +1,255 @@
+      params:
+        - name: git_username
+          value: $(params.git_username)
+        - name: git_pr_title
+          value: $(params.git_pr_title)
+        - name: pyxis_url
+          value: $(params.pyxis_url)
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    # reserve operator name
+    - name: reserve-operator-name
+      runAfter:
+        - submission-validation
+      taskRef:
+        name: reserve-operator-name
+      params:
+        - name: operator_name
+          value: "$(tasks.submission-validation.results.operator_name)"
+
+    # additional checks
+    - name: operator-validation
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: operator-validation
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    - name: yaml-lint
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: yaml-lint
+      params:
+        - name: args
+          value: ["-d {extends: relaxed, rules: {line-length: {max: 180}}}", "$(params.bundle_path)"]
+      workspaces:
+        - name: shared-workspace
+          workspace: repository
+          subPath: src
+
+    - name: digest-pinning
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: digest-pinning
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    - name: verify-pinned-digest
+      runAfter:
+        - digest-pinning
+      taskRef:
+        name: verify-pinned-digest
+      params:
+        - name: dirty_flag
+          value: "$(tasks.digest-pinning.results.dirty_flag)"
+
+    - name: content-hash
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: content-hash
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    - name: verify-changed-directories
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: verify-changed-directories
+      params:
+        - name: operator_name
+          value: "$(tasks.submission-validation.results.operator_name)"
+        - name: operator_version
+          value: "$(tasks.submission-validation.results.operator_version)"
+        - name: git_pr_branch
+          value: $(params.git_pr_branch)
+
+    - name: query-publishing-checklist
+      runAfter:
+        - reserve-operator-name
+      taskRef:
+        name: query-publishing-checklist
+      params:
+        - name: cert_project_id
+          value: "$(tasks.submission-validation.results.cert_project_id)"
+
+    # Try to retrieve results
+    - name: get-ci-results1
```

Comment:
Let's find a better name for the task without a sequence number as a suffix.

Reply:
get-ci-results-attempt and get-ci-results?